test/cli/detect_breakends_cli_test.cpp

```diff
@@ -59,9 +59,14 @@ const std::string help_page_part_2
 
 const std::string help_page_advanced
 {
-    "    -m, --method (List of unsigned 8 bit integer)\n"
-    "          Choose the detecting method(s) to be used. Default: [1,2,3,4]. Value\n"
-    "          must be in range [1,4].\n"
+    "    -m, --method (List of detection_methods)\n"
+    "          Choose the detection method(s) to be used. Default:\n"
+    "          [cigar_string,split_read,read_pairs,read_depth]. Value must be one\n"
+    "          of\n"
+    "          [read_depth,read_depth,read_pairs,read_pairs,split_read,split_read,cigar_string,cigar_string].\n"
```

Comment:
Why is each method included twice?

Reply:
We don't use the new Validator for the detection methods enum, because I got a bug in a test. I opened an issue which should solve this problem: https://github.com/seqan/iGenVar/issues/78.